connector.go

```diff
@@ -64,6 +68,21 @@ func applyConnector(c *cli.Context) error {
 		content = replacePlaceholderValues(content, REGION_NAME_PLACEHOLDER, awsRegion)
 		content = replacePlaceholderValues(content, DELEGATE_NAME_PLACEHOLDER, delegateName)
 	}
+	if isPdcConnectorYAML(content) {
+		hasPortNumber := strings.Contains(content, HOST_PORT_PLACEHOLDER)
+		if hostIpOrFqdn == "" || hostIpOrFqdn == HOST_IP_PLACEHOLDER {
+			githubUsername = TextInput("Enter valid host ip / fqdn:")
+		}
+		if hasPortNumber && (hostPort == "" || hostPort == HOST_PORT_PLACEHOLDER) {
+			githubUsername = TextInput("Enter valid host port:")
```

Comment:
here the `githubUsername` variable is being reused for different inputs (host IP/FQDN and host port), distinct variable name would be better.

Reply:
it was a miss updated in recent commit